src/tiles/routes/tilesRouter.ts

```diff
@@ -7,6 +7,7 @@ const tilesRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
   const controller = dependencyContainer.resolve(TilesController);
```

Comment:
export both the router and the symbol in the same fashion

Reply:
fixed